sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSourceStrategy.scala

```diff
@@ -631,6 +631,19 @@ object DataSourceStrategy
     }
   }
 
+  protected[sql] def translateRuntimeFilter(expr: Expression): Option[Filter] = expr match {
```

Comment:
Super minor suggestion, we could maybe add some scaladoc here on the expected behaviour, I note that in one case it rases an exception and in the other case it returns None to indicate it isn't able to handle the filter and I think clarifying that could be good.

Reply:
Will add.

Reply:
Added. Let me know if it is descriptive enough, @holdenk.